packages/now-cli/src/util/config/global-path.ts

```diff
@@ -1,12 +1,36 @@
 // Native
 import { homedir } from 'os';
 
+import fs from 'fs';
 import path from 'path';
 
 // Packages
 import mri from 'mri';
+import xdg from 'xdg-portable';
+import head from 'lodash.head';
 
-const getNowDir = () => {
+// The `homeConfigPath` is the legacy configuration path located in the users home directory.
+const homeConfigPath: string = path.join(homedir(), '.now');
+
+// The `xdgConfigPath` is the configuration path which is based on the XDG standard.
+// the old legacy path is used as a fallback
+const xdgConfigPath: string = path.join(
+  head(xdg.dataDirs()) || homeConfigPath,
```

Comment:
Is lodash really necessary here? Why not just `xdg.dataDirs()[0]`?

```suggestion
  xdg.dataDirs()[0] || homeConfigPath,
```

Reply:
I've removed the lodash dependency and used your suggested change.